src/app/page.js

```diff
@@ -6,7 +6,7 @@ import Upcomming from '@/components/home/Upcomming';
 
 export default function Home() {
   return (
-    <main>
+    <main className='min-h-[120vh]'>
```

Comment:
Why do we need a minimum height 120vh in our main tag?

Reply:
Yes, that was for testing the sticky header. I forgot to remove it. Thanks for pointing out.